taskt/Core/Automation/User32/GlobalHook.cs

```diff
@@ -0,0 +1,639 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Windows.Forms;
+using taskt.Core.Automation.Commands;
+
+namespace taskt.Core.Automation.User32
+{
+    public class GlobalHook
+    {
+        private const int WhKeyboardLl = 13;
+        private const int WmKeyDown = 0x0100;
+        private static readonly LowLevelKeyboardProc _kbProc = KeyboardHookEvent;
+        private static readonly LowLevelMouseProc _mouseProc = MouseHookEvent;
+        private static readonly LowLevelMouseProc _mouseLeftUpProc = MouseHookForLeftClickUpEvent;
+        private static IntPtr _keyboardHookID = IntPtr.Zero;
+        private static IntPtr _mouseHookID = IntPtr.Zero;
+        private static Stopwatch _sw;
```

Comment:
Can you rename this something meaningful? Like _stopWatch.

Reply:
done